src/i18n/es-ES/index.ts

```diff
@@ -1,18 +1,27 @@
 export default {
   language: 'Idioma',
   ddos: {
-    description: 'Ataque DDOS que realiza muchas requests/solicitudes a targets de servidores enemigos. Los targets/objetivos comenzarán a funcionar mal o con lag.',
+    description: 'Ataque DDoS que realiza muchas requests/solicitudes a targets de servidores enemigos. Los targets/objetivos comenzarán a funcionar mal o con lag',
+    coordinators: 'Coordinación:: https://t.me/incourse911',
     counter: {
       attackedTimes: 'Número de ataques:',
-      currentTarget: 'Target/Objetivo actual:'
+      currentTarget: 'Target/Objetivo actual: '
     },
     enable: {
-      name: 'Ataque DDOS',
-      description: 'Habilitar ataque DDOS en servidores críticos enemigos: gobierno, sistemas bancarios, webs de propaganda, ...'
+      name: 'Ataque DDoS',
+      description: 'Habilitar ataque DDoS en servidores críticos enemigos: gobierno, sistemas bancarios, webs de propaganda, ...'
     },
     proxy: {
       name: 'Atacar sólo usando proxy',
       description: 'Sin esta opción, el programa permitirá usar tu IP. Realizarás ataques más eficientes, pero los ataques no serán anónimos.'
+    },
+    advanced: {
```

Comment:
@belueg could you help with translating this to Spanish?

Reply:
I just wrote here in the PR the translation, but if you need me to open a new PR, no problem